src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/Interop/MsQuicNativeMethods.cs

```diff
@@ -299,58 +446,68 @@ internal struct ConnectionEvent
             IntPtr context,
             ref ConnectionEvent connectionEvent);
 
+        // TODO: order is Open, Close, Shutdown, Start, SetConfiguration, SendResumptionTicket
```

Comment:
Can this be removed now?

Reply:
Nope, that's my comment. The order (in the comment) is taken from msquic.h, we do not follow that in here. I didn't want to generate more diffs in this PR. I'll plan to do follow up that does not change the behavior at all, just shuffles things around to be more like msquic.h.